crates/stackable-operator/CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ All notable changes to this project will be documented in this file.
 
 - Re-export the `YamlSchema` trait and the `stackable-shared` crate as the `shared` module ([#883]).
 - BREAKING: Added `preferredAddressType` field to ListenerClass CRD ([#885]).
+- The cluster domain (default: `cluster.local`) can now be configured in the individual operators 
+  via the ENV variable `KUBERNETES_CLUSTER_DOMAIN` or resolved automatically by parsing the 
+  `/etc/resolve.conf` ([#893]).
```

Comment:
Please mention the breaking change that we renamed create_client to initialize_operator

Reply:
done https://github.com/stackabletech/operator-rs/pull/893/commits/e053479a2fe011eed64a6c646283ec0c056fca4d